README.md

```diff
@@ -130,8 +125,7 @@ A minimal working example of a program that uses the library is:
 
 		MyDistanceCallback d;
 
-		TapkeeOutput output = tapkee::initialize()
-		   .withParameters((method=MultidimensionalScaling,target_dimension=1))
+		TapkeeOutput output = tapke::with((method=MultidimensionalScaling,target_dimension=1))
```

Comment:
Oh there is a typo

Reply:
Is it the indentation? In that case, yeah I see that maybe this line is a bit longer than the other ones with code (here in the inline diff it looks, while it is only one). Any option looks go to me, like it is with the long-ish line, or, alternatively, finishing this line at `=` and aligning the `tapkee::with` with the other `with*` below.

I tried but didn't manage to find a typo in the names in this line ':-D

Reply:
No it is `tapke`

Reply:
Ah, finally see it, thank you!